04_verify.sh

```diff
@@ -22,40 +22,40 @@ fi
 check_bm_hosts() {
     local FAILS_CHECK="${FAILS}"
     local NAME ADDRESS USER PASSWORD MAC CRED_NAME CRED_SECRET \
-      BM_HOSTS BM_HOST BM_VMS BM_VMNAME BM_VM_IFACES
+      BARE_METAL_HOSTS BARE_METAL_HOST BARE_METAL_VMS BARE_METAL_VMNAME BARE_METAL_VM_IFACES
```

Comment:
Rather than making it an extra long line, just have two lines of `local ...`

Reply:
done